HISTORY.rst

```diff
@@ -43,6 +43,8 @@ Unreleased Changes (3.9)
       ``projection_wkt``.
   * Fixed bug that was causing a TypeError when certain input rasters had an
     undefined nodata value. Undefined nodata values should now work everywhere.
+  * Use consistent naming for geopackage (.gpkg) output files and layers. Any
+    suffix applied to the file name will also be applied to the layer name.
```

Comment:
In part because this only affects a couple of models, could you also make a note under each of the affected model's sections indicating which layer was renamed, and what it was renamed to?  I'm imagining future us wondering why some script broke and trying to find out when the layer name changed to something completely different!

Reply:
You're right, looking back on it this isn't a very good explanation. I added a note to each affected model